vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_event_unittest_util.cc

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_event_unittest_util.h"
+
+#include <string>
+#include <vector>
+
+#include "base/base64.h"
+#include "base/guid.h"
+#include "bat/ads/internal/base/crypto/crypto_util.h"
+#include "bat/ads/internal/base/unittest/unittest_constants.h"
+#include "bat/ads/internal/ml/data/vector_data.h"
+
+namespace ads {
+
+ml::pipeline::TextEmbeddingInfo BuildTextEmbeddingInfo() {
+  const std::string text = base::GUID::GenerateRandomV4().AsLowercaseString();
```

Comment:
`text` is confusing as it is a `GUID`

Reply:
@tmancey GUID is only used here to generate a random string. Is there a different base function I could use to easily generate a random string?